packages/nodes-base/nodes/Airtop/actions/interaction/scroll.operation.ts

```diff
@@ -0,0 +1,224 @@
+		displayOptions: {
+			show: {
+				resource: ['interaction'],
+				operation: ['scroll'],
+				scrollingMode: ['automatic'],
+			},
+		},
+	},
+	{
+		displayName: 'Scroll To Page Edges',
+		name: 'scrollToEdge',
+		type: 'fixedCollection',
+		default: {},
+		placeholder: 'Add Edge Direction',
+		description:
+			"The direction to scroll to. When 'Scroll By' is defined, 'Scroll To Edge' action will be executed first, then 'Scroll By' action.",
+		displayOptions: {
+			show: {
+				resource: ['interaction'],
+				operation: ['scroll'],
+				scrollingMode: ['manual'],
+			},
+		},
+		options: [
+			{
+				displayName: 'Page Edges',
+				name: 'edgeValues',
+				values: [
+					{
+						displayName: 'Vertically',
+						name: 'yAxis',
+						type: 'options',
+						default: '',
+						options: [
+							{
+								name: 'Empty',
+								value: '',
+							},
+							{
+								name: 'Top',
+								value: 'top',
+							},
+							{
+								name: 'Bottom',
+								value: 'bottom',
+							},
+						],
+					},
+					{
+						displayName: 'Horizontally',
+						name: 'xAxis',
+						type: 'options',
+						default: '',
+						options: [
+							{
+								name: 'Empty',
+								value: '',
+							},
+							{
+								name: 'Left',
+								value: 'left',
+							},
+							{
+								name: 'Right',
+								value: 'right',
+							},
+						],
+					},
+				],
+			},
+		],
+	},
+	{
+		displayName: 'Scroll By',
+		name: 'scrollBy',
+		type: 'fixedCollection',
+		default: {},
+		description:
+			"The amount to scroll by. When 'Scroll To Edge' is defined, 'Scroll By' action will be executed after 'Scroll To Edge'.",
+		placeholder: 'Add Scroll Amount',
+		displayOptions: {
+			show: {
+				resource: ['interaction'],
+				operation: ['scroll'],
+				scrollingMode: ['manual'],
+			},
+		},
+		options: [
+			{
+				name: 'scrollValues',
+				displayName: 'Scroll Values',
+				description: 'The amount in pixels or percentage to scroll by',
+				values: [
+					{
+						displayName: 'Vertically',
+						name: 'yAxis',
+						type: 'string',
+						default: '',
+						placeholder: 'e.g. 200px, 50%, -100px',
+					},
+					{
+						displayName: 'Horizontally',
+						name: 'xAxis',
+						type: 'string',
+						default: '',
+						placeholder: 'e.g. 50px, 10%, -200px',
+					},
+				],
+			},
+		],
+	},
+	{
+		displayName: 'Scrollable Area',
+		name: 'scrollWithin',
+		type: 'string',
+		default: '',
+		description: 'Scroll within an element on the page',
+		placeholder: 'e.g. the left sidebar',
+		displayOptions: {
+			show: {
+				resource: ['interaction'],
+				operation: ['scroll'],
+				scrollingMode: ['automatic'],
+			},
+		},
+	},
+];
+
+export async function execute(
+	this: IExecuteFunctions,
+	index: number,
+): Promise<INodeExecutionData[]> {
+	const { sessionId, windowId } = validateSessionAndWindowId.call(this, index);
+
+	const scrollingMode = validateScrollingMode.call(this, index);
+	const isToEdges = scrollingMode === 'toEdges';
```

Comment:
isn't scrolling mode either `automatic` or `manual`? Shouldn't this be `manual`? Also correct the comments on line 204 and 207.
```suggestion
	const isToEdges = scrollingMode === 'manual';
```